### DISCUSSION TITLE
createCellBelow not honoring source

### DISCUSSION THREAD

**Discussion by emeeks** [nteract/archived-desktop-app#5286] [Q&A] [answered]
**Application or Package Used**
@nteract/core

**Describe the bug**
When I fire an `actions.createCellBelow({ id: validid, cellType: "code", source: "Some kind of set source", contentRef: validContentRef })` the cell is created properly but is empty.

**To Reproduce**
fire `actions.createCellBelow` with a string in source

**Comment by captainsafia** [comment, marked as answer]
You can try passing a `cell` property to the `createCellBelow` action with the cells content, similar to the code below.

https://github.com/nteract/nteract/blob/f8afe2940b4b44838b29da5a9016e73e134c5bd6/applications/desktop/src/notebook/insert-images.ts#L89-L98
